h2/events.py

```diff
@@ -334,9 +334,18 @@ def __init__(self):
         #: peer and so can safely be resent.
         self.last_stream_id = None
 
+        #: Additional debug data that can be appended to GOAWAY frame.
+        self.additional_data = None
+
     def __repr__(self):
-        return "<ConnectionTerminated error_code:%s, last_stream_id:%s>" % (
-            self.error_code, self.last_stream_id
+        return (
+            "<ConnectionTerminated error_code:%s, last_stream_id:%s, "
+            "additional_data:%s>" % (
+                self.error_code,
+                self.last_stream_id,
+                (_bytes_representation(self.additional_data[:20])
+                 if self.additional_data else None)
```

Comment:
Heh, yeah, good spot: I missed that we sliced first. Yeah, ok, let's just adjust the indentation as you suggest.